3main

```diff
@@ -934,7 +934,7 @@ then
 fi
 
 # Apply GPU settings
-sudo ${NVD} $NVD_SETTINGS
+sudo ${NVD} $NVD_SETTINGS &
```

Comment:
Let it be this way, we will change it after public beta testing if it caused problems

Reply:
1) Would it be a problem if clocks are applied after miner is started?
2) I'm not so sure the time required still depends on the number of installed gpus

Reply:
Dont know, thought it may cause problem when gpu is under heavy load and OC settings changes. 
As I said earlier lets see how it goes in beta testing.